scheduler/numa_ce.go

```diff
@@ -28,8 +30,13 @@ type coreSelector struct {
 func (cs *coreSelector) Select(ask *structs.Resources) ([]uint16, hw.MHz) {
 	cores := cs.availableCores.Slice()[0:ask.Cores]
 	mhz := hw.MHz(0)
+	sortedTopologyCores := make([]numalib.Core, len(cs.topology.Cores))
+	copy(sortedTopologyCores, cs.topology.Cores)
+	slices.SortFunc(sortedTopologyCores, func(a, b numalib.Core) int { return cmp.Compare(a.ID, b.ID) })
```

Comment:
Let me know what you think about leaving it like this, or if it's better to rework how the cores are stored in the `struct`.

Reply:
This looks fine; we want to keep that data structure the same due to requirements in the enterprise version.